tensorflow/c/c_api.h

```diff
@@ -101,6 +101,11 @@ typedef enum {
 } TF_DataType;
 
 // --------------------------------------------------------------------------
```

Comment:
I wouldn't have a divider here -- this is closely related to the TF_DataType definition.